pkg/storage/iterator_test.go

```diff
@@ -680,3 +807,25 @@ func TestDropLabels(t *testing.T) {
 		})
 	}
 }
+
+func Test_IsInvalidChunkError(t *testing.T) {
+	tests := []struct {
+		name string
+		err  error
```

Comment:
You need to add a case at least where it is not an invalid chunk error.

Reply:
added